Travel_fellowships.md

```diff
@@ -144,13 +141,62 @@ This will not be a limiting factor if applicants are new to bioinformatics and o
+
+## Rubric for Application Review
+
+We ask reviewers to evaluate the application across multiple dimensions with scores of 1, 2, or 3 that they share via a Google form with the Event Fellowship chair. 
+These correspond generally to reviews of "criteria not met", "criteria partially met" and "criteria fully met":
+- Applicants who score mostly 1s do not have a clear understanding of the overall goals of this fellowship to support Open Source software practices and diversity of attendees in bioinformatics/biological research/open science events and communities.
+- Applicants who score mostly 2s are enthusiastic if not wholly aligned with the event fellowship's goals, for example, they may not be attending an open-source related event but have an interest in learning and promoting best practices, or that they want to attend a suitable event but do not represent any underrepresented minority groups.
+- Applicants who score mostly 3s are ready, goal-oriented, interested in contributing to the open-source, and represent traditionally underrepresented or minority groups in open science/bioinformatics.
+
+Reviewers are also asked whether they would recommend an application. The available answers are "No", "Yes", "Unsure" or "Request to discuss". Candidates who have ratings of "No" or "Unsure" across all reviewers will be rejected. All other candidates will be discussed at the panel selection.
+
+We ask all reviewers to finish the review with a few summary sentences to describe the application and the motivations for their scores to facilitate conversation during the selection panel discussion.
+Below we provide a copy of our review form.
+
+### Event and nature of applicant's participation
+
+- [ ] 1 (criteria not met) Does not provide enough information or seems to misunderstand what kind of event OBF Event fellowship supports
+- [ ] 2 (criteria partially met) The event or the nature of their participation may not directly relate to Open Source in bioinformatics, but they have some understanding/plan on how they would use the knowledge gained at this event to promote open source practices and diversity
+- [ ] 3 (criteria fully met) Seems to have a clear understanding of how their participation in this event relates to open-source bioinformatics or open science in general, and how they will further use and promote the knowledge obtained at this event
+
+### Open Source Resources
+- [ ] 1 (criteria not met) Shares none or unrelated/irrelevant information on open source resource and how they relate to their proposed event
+- [ ] 2 (criteria partially met) Shares some relevant information on open source resource and how they relate to their proposed event
+- [ ] 3 (criteria fully met) Shares appropriate and relevant information on open source resource and how they relate to their proposed event
+
+### Promoting diversity and representation of historically excluded or underrepresented groups *
```

Comment:
Why the * here?

Reply:
```suggestion
### Promoting diversity and representation of historically excluded or underrepresented groups
```

Reply:
That was copy paste error (Google form uses * for required section)